### PR TITLE
AAdded project relaed files

### PR DESCRIPTION
**Pull request by dmnglore**
AAdded project relaed files

### CODE REVIEW

File: mvnw
Comment (sadhanmb) [inline mvnw:1]: delete codeowners.txt which is a unused file

File: src/test/java/com/craftdmo/githubdemo/GithubdemoApplicationTests.java
Comment (sadhanmb) [inline src/test/java/com/craftdmo/githubdemo/GithubdemoApplicationTests.java:1]: Add more tests for the scenarios

File: src/main/resources/application.properties
Comment (sadhanmb) [inline src/main/resources/application.properties:1]: provide unique port number since 8080 might be used by some other application

**Review by sadhanmb — CHANGES_REQUESTED** [review]
please validate all review comments and take actions accordingly

**Comment by dmnglore** [comment]
> please validate all review comments and take actions accordingly

Fixed

**Comment by dmnglore** [comment]
fxed

File: mvnw
Comment (dmnglore) [reply mvnw:1]: fixed

**Comment by dmnglore** [comment]
addressed all review commens

File: mvnw
Comment (dmnglore) [reply mvnw:1]: fixed

File: src/test/java/com/craftdmo/githubdemo/GithubdemoApplicationTests.java
Comment (dmnglore) [reply src/test/java/com/craftdmo/githubdemo/GithubdemoApplicationTests.java:1]: added

**Comment by dmnglore** [comment]
merged all changes

**Pull request merged**